hypex/selectors/feature_selector.py

```diff
@@ -84,19 +57,21 @@ def perform_selection(self, df: pd.DataFrame) -> pd.DataFrame:
             "drop": [self.treatment],
         }
 
-        if self.outcome_type == "numeric":
-            task_name = "reg"
-            loss = "mse"
-            metric = "mse"
-        elif self.outcome_type == "binary":
-            task_name = "binary"
-            loss = "logloss"
-            metric = "logloss"
+        if self.use_algos == 'lgb':
+            selector = selector_primal_methods.pd_lgbm_feature_selector
+        elif self.use_algos == 'catboost':
+            selector = selector_primal_methods.pd_catboost_feature_selector
+        elif self.use_algos == 'ridgecv':
+            selector = selector_primal_methods.pd_ridgecv_feature_selector
         else:
-            task_name = "multiclass"
-            loss = "crossentropy"
-            metric = "crossentropy"
-
-        features_scores = []
-
-        return features_scores
+            raise Exception(f"Unknown input algorithm used on feature_selector: {self.use_algos}")
```

Comment:
Maybe it would be better to assign USE_ALGOS when the name of the algorithm is entered incorrectly? And throw out a warning
Otherwise, the whole pipeline will collapse in the future.

Reply:
In `__init__` for Example.

Reply:
And get selectors from dict.

Reply:
Check this in `__init__` and from `dict` Instead of `if-else`.